app/services/twilio_voice_service.rb

```diff
@@ -0,0 +1,13 @@
+class TwilioVoiceService
+  attr_accessor :api
+
+  def initialize
+    @service = TwilioService.new
+    @api = @service.client
+  end
+
+  def place_call(params = {})
+    params = params.reverse_merge(from: @service.from_number)
+    @api.calls.create(params)
```

Comment:
Why not just define both `place_call` and `send_sms` on `TwilioService`?   Seems like it would be simpler to follow.